lib/streamlit/elements/media.py

```diff
@@ -79,6 +81,10 @@ def audio(
         sample_rate: int or None
             The sample rate of the audio data in samples per second. Only required if
             ``data`` is a numpy array.
+        end_time: int
+            The time at which this element should stop playing.
```

Comment:
The smallest reasonable measure to configure a video/audio start/end time is probably seconds or? So no need to use `float` here?

Reply:
> The smallest reasonable measure to configure a video/audio start/end time is probably seconds or? So no need to use float here?

@LukasMasuch The `currentTime` property of the `HTMLMediaElement` interface from which `HTMLVideoElement` inherits is specified in seconds according to the docs: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/currentTime

And it can be set or returned as "_a double-precision floating-point value indicating the current playback time in seconds_", so we can support `float` here. Should we do it in this PR or another one scoped specifically for this change?

Reply:
> we also support timedelta and str (parsed by Pandas) in addition to the number of seconds. Maybe also something to consider here

@LukasMasuch really good point! This is a suggestion Thiago also had when we presented these specs in office hours.

Supporting `timedelta` objects from [datetime](https://docs.python.org/3/library/datetime.html#timedelta-objects) and a string specifying the time in a format supported by [Pandas's Timedelta constructor](https://pandas.pydata.org/docs/reference/api/pandas.Timedelta.html), in addition to seconds, would be nice for consistency across the API and will reduce the learning curve for new users. 

+1 to supporting them.